txpipe/twopoint_null_tests.py

```diff
@@ -481,13 +482,13 @@ class TXGammaTRandoms(TXTwoPoint):
         "min_sep": 2.5,
         "max_sep": 100,
         "nbins": 20,
-        "bin_slop": 0.1,
+        "bin_slop": 1,
         "sep_units": "arcmin",
         "flip_g1": False,
         "flip_g2": True,
         "cores_per_task": 20,
         "verbose": 1,
-        "reduce_randoms_size": 1.0,
+        "reduce_randoms_size": 100.0,
```

Comment:
Sorry - one last thing I spotted - should this reduction be so large?  Fine if there is a good reason, just wanted to check.

Reply:
I think this was just to make it run faster to get through it quickly. Let me revert it